app/lib/features/pins/pages/create_pin_page.dart

```diff
@@ -294,6 +294,7 @@ class _CreatePinConsumerState extends ConsumerState<CreatePinPage> {
                     ? RenderHtml(
                       text: params.htmlBodyDescription,
                       defaultTextStyle: textTheme.labelLarge,
+                      roomId: '',
```

Comment:
Why not making it optional rather than passing empty ?

Reply:
as stated in the todos: that's the plan.